service/web/tabs/app/src/utilities/theme.ts

```diff
@@ -11,13 +11,10 @@ const environmentColorLookup: IEnvironmentToColorLookup = {
   DEVELOPMENT: color.platinum,
   TESTING: color.purple,
   STAGING: color.yellow,
-  PRODUCTION: color.red
+  PRODUCTION: "#db2636"
 }
 
-export const backfilaRed = "#db2636"
-
 export const backfilaTheme: ITheme = {
   ...defaultTheme,
-  environmentToColor: environmentToColor(environmentColorLookup),
-  navbarBackground: backfilaRed
```

Comment:
Have you tested this locally? navbarBackground currently applies across all environments so this change would remove the red from the navbar even in production. Making navbar background environment specific is easily possible but would require associated Misk-Web changes.

Reply:
Ok, I'm fine dropping the "backfilaRed" I think the stylistic quirkiness of that is less important than accurately reflecting environments as a visual cue

Reply:
Instead of the suggested changes, can you remove the theme override in LayoutContainer and leave this file as is? https://github.com/cashapp/backfila/blob/229b39eddc60d6b0f4d735ffebb177f061bb21bf/service/web/tabs/app/src/containers/LayoutContainer.tsx#L19

Reply:
Just changed this to use the default misk-web behavior w.r.t. navbarBackground.
My read of misk-web is that it will use `environmentToColor` as defined by the theme:

```
packages/@misk/core/src/features/Navbar/Banner.tsx
24:  background-color: ${theme.environmentToColor(environment)} !important;
```

Reply:
Gonna decline this since there's a potentially better fix in misk-web that will obviate changes in backfila itself